indico/modules/events/editing/client/js/components/Timeline/judgment/UpdateFilesForm.jsx

```diff
@@ -78,7 +92,21 @@ export default function UpdateFilesForm({setLoading}) {
             <FinalTagInput name="tags" options={staticData.tags} />
           </Form>
           <div styleName="judgment-submit-button">
-            <FinalSubmitButton form="judgment-form" label={Translate.string('Confirm')} />
+            <Button.Group color="blue">
+              <FinalSubmitButton form="judgment-form" label={option.text} />
+              <Dropdown
+                className="button icon"
+                form="judgment-form"
+                text={null}
+                button
+                floating
+                disabled={hasValidationErrors || pristine || submitting}
+                options={confirmOptions}
+                onChange={(e, data) => setConfirmationType(data.value)}
+                // eslint-disable-next-line react/jsx-no-useless-fragment
+                trigger={<Fragment />}
```

Comment:
`trigger` - Custom element to trigger the menu to become visible. Takes place of 'text'.
I tried to supply `{null}` to avoid the use of `Fragment` but it still shows in the dropdown.